src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -67,7 +67,9 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
             <RenderMarkdown text={event.description} />
           </div>
         </div>
+
         {isCurrentEvent && (
+          //  @debt Does ScheduleItem__entry-room-button even exist? Can we remove this?
```

Comment:
It does exist. It appears only for Live events.
![image](https://user-images.githubusercontent.com/63313732/123519794-62b0af80-d67b-11eb-91e3-808246eadf09.png)

But I've just tested. When you click this button it opens in the two tabs: current and a new one. 😬 
So looks like we can remove the `@debt` comment and `externalUrlAdditionalProps` from here. 🙃

Reply:
also. Might be worth renaming the variable `isCurrentEvent` -> `isLiveEvent` ?

Reply:
Could there be some ambiguity  if just `isLiveEvent`?

Then maybe something like `isEventLiveNow`, `isEventOngoingATM`...

Reply:
@goran-peoski-work Not really. An event is live or it isn’t. That aligns to our other current naming, including related helpers.

Reply:
@sunny-viktoryia i’ll have to check again, but I’m sure I searched for the code and couldn’t find its definition, which is why I added the debt comment. I’ll check again.

I’ll also check the bug you mention here and fix it up too.

Reply:
Ok, just confirmed, I was searching for the full class name, but obviously because we use `&` I needed to search for `__entry-room-button`. Can confirm that it exists, will remove the `@debt`

Reply:
> But I've just tested. When you click this button it opens in the two tabs: current and a new one. 😬
> So looks like we can remove ... externalUrlAdditionalProps from here. 🙃

So this is likely happening because there is both the `onClick={onRoomEnter}` as well as the `href={roomUrl}`.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1657/commits/fe21a57ae1a329c3ef583fbd123d57a73f9dbfea by using `e.preventDefault()`

- https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault
  - > The `Event` interface's `preventDefault()` method tells the user agent that if the event does not get explicitly handled, its default action should not be taken as it normally would be. The event continues to propagate as usual, unless one of its event listeners calls `stopPropagation()` or `stopImmediatePropagation()`, either of which terminates propagation at once.
  
Basically, this says "when the `onClick` handler runs, don't also let the `<a href>`'s default run, and thus will prevent opening 2 windows.

Note that there are also a few nuances around using `<a href>` with regards to opening rooms/etc, which could impact user presence/counting. I've written more about it in https://github.com/sparkletown/sparkle/issues/1670

Reply:
bonus: fixed 'onend' not firing in `useCustomSound` when the sound sprite is missing from the config in https://github.com/sparkletown/sparkle/pull/1657/commits/8820af929165bc9e340a1a2b9b18965e7a3e8db3 (which just happened to occur on the room I picked to test all of this on, and was stopping me from being able to enter

Reply:
> Might be worth renaming the variable `isCurrentEvent` -> `isLiveEvent` ?

> Could there be some ambiguity if just isLiveEvent?
> 
> Then maybe something like isEventLiveNow, isEventOngoingATM...

fixed in https://github.com/sparkletown/sparkle/pull/1657/commits/f909b4868fa06bf9de677a88cf7363f53139886e by removing the `isCurrentEvent` prop from `ScheduleItem` entirely, and just calculating it within the component itself. Now called `isCurrentEventLive` to disambiguate it from the function from `utils/time` which is named `isEventLive`

Reply:
bonus fix: renamed `ScheduleItem`'s 'passively named' `onRoomEnter` prop to the 'active action named' `enterEventLocation` in https://github.com/sparkletown/sparkle/pull/1657/commits/9e445c8601c73da181b4c962d8cc13b5f8094c3c